packages/core/src/components/label/label.tsx

```diff
@@ -10,6 +10,9 @@ export class Label {
   @Prop()
   status?: 'primary' | 'info' | 'success' | 'warning' | 'danger';
 
+  @Prop()
+  small?: boolean;
```

Comment:
Zoals `small` hier is ingezet sluit deze niet aan op de huidige toepassing van de term "small". Small geeft een schaal aan: Small, medium, large. Daarnaast zijn het aliassen voor vaste maten.

In deze context is er geen andere aanduiding dan "small". Ik stel voor dat we dit gaan bestempelen als `compact`, wat jij?

Edit: Dit kan persoonlijk zijn, maar ik vind dat `small` op het web iets over _horizontale_ maten zegt (responsive web design), maar ik lees verderop in de code dat alleen de _verticale_ padding wordt aangepast

Reply:
Ik vind `compact` inderdaad een betere term, dan stel ik voor om ook de class aan te passen naar `dso-compact`